### DISCUSSION TITLE
This piece of code, breaks the layout

### DISCUSSION THREAD

**Discussion by johanroug** [cssbattle/forum#392] [Bugs reports]
If I write this piece of code, the layout breaks
* {
    display: flex;
  }

![image](https://github.com/cssbattle/forum/assets/1043773/40fffbeb-dbaf-4c6a-abf3-30bf8e9ac18e)

**Comment by chinchang** [comment]
Its not actually breaking. When you make everything (*) `display: flex`, it affects the `<style>` tag also. `<style>` tags are `display: none` by default, but when you make them `display: flex`, their content starts showing :)
Wonderful CSS :)

**Reply by SelenIT** [reply]
It's also applied to `html`, `head`, and `body` elements that are always present in any HTML page regardless the presence of respective tags.
